### DISCUSSION TITLE
i can't find theme folder. Help pls !!!!

### DISCUSSION THREAD

**Discussion by hoanglong2206** [primefaces/community#2834] [PrimeNG]
![Screenshot 2024-10-10 152151](https://github.com/user-attachments/assets/019939f7-41a0-49f8-a7cf-b805e3bc5f56)

**Comment by qwadrox** [comment]
Hi @hoanglong2206! 

The import is correct. Could you let me know if you installed the correct version of PrimeNg? It should be V18, not V17.
```ts
import { Aura } from 'primeng/themes/aura';
``` 
After making sure you installed the npm dependencies, you might need to restart the TypeScript language server sometimes because VScode can be laggy.

**Reply by hoanglong2206** [reply]
![Screenshot 2024-10-12 231317](https://github.com/user-attachments/assets/7772738c-3ca3-4e2e-b10b-7f685f35e2e6)
i use this version

**Reply by qwadrox** [reply]
This is the Angular cli version and not the Primeng version. 
Follow this: https://primeng.org/installation
Your package.json should contain Primeng. 
Make sure you use the correct version of Primeng. In your case you have to install v18.
 
https://www.npmjs.com/package/primeng?activeTab=versions

But I advice you to use V17 because V18 is still beta.
